packages/stable/src/api/vision/types.gen.ts

```diff
@@ -55,77 +72,134 @@ export interface AnnotationsBoundingBox {
   yMin: number;
 }
 /**
-* A point attached with additional information such as a confidence value and
-various attribute(s).
-*/
+ * primitives.geometry2d.Keypoint
+ * A point attached with additional information such as a confidence value and
+ * various attribute(s).
+ */
 export interface AnnotationsKeypoint {
-  /** Additional attributes data for a compound. */
-  attributes?: Record<string, AnnotationsBoolean | AnnotationsNumerical>;
+  /**
+   * Additional attributes data for a compound.
+   * @default null
+   */
+  attributes?: Record<
+    string,
+    | ({
```

Comment:
this looks kind of weird?

Reply:
agree this seems broken (initially was correct)

Reply:
@andersfylling can we override this type somehow?

Reply:
I'll have a look again later today